templates/terraform/constants/health_check.erb

```diff
@@ -0,0 +1,77 @@
+// Whether the port should be set or not
+func validatePortSpec(diff *schema.ResourceDiff, blockName string, defaultPort int) error {
```

Comment:
I'm not sure default port is getting used. I don't think we need it since this is just making sure the port is not set, but please correct me if I misread!